assets/schema_input.json

```diff
@@ -9,115 +9,139 @@
         "properties": {
             "test_vcf": {
                 "type": "string",
+                "exists": true,
                 "pattern": "\\S+\\.vcf(\\.gz)?$",
                 "format": "file-path",
-                "errorMessage": "Test VCF must be provided, cannot contain spaces and must have extension '.vcf.gz'"
+                "errorMessage": "Test VCF must be provided, cannot contain spaces and must have extension '.vcf.gz'",
+                "description": "The VCF file to use as benchmarking test input"
             },
             "id": {
                 "type": "string",
                 "pattern": "^\\S+$",
-                "errorMessage": "Sample id",
-                "meta": ["id"]
+                "description": "Sample id",
+                "meta": ["id"],
+                "minLength": 1
             },
             "caller": {
                 "type": "string",
                 "pattern": "^\\S+$",
-                "errorMessage": "Name of the variant caller used to generate test file",
-                "meta": ["caller"]
+                "description": "Name of the variant caller used to generate test file",
+                "meta": ["caller"],
+                "minLength": 1
             },
             "vartype": {
                 "type": "string",
                 "pattern": "^\\S+$",
-                "errorMessage": "Variant type to benchmark",
-                "meta": ["vartype"]
+                "description": "Variant type to benchmark",
+                "meta": ["vartype"],
+                "minLength": 1
             },
             "subsample": {
                 "type": "string",
                 "pattern": "^\\S+$",
-                "errorMessage": "Name to subsample in the multisample VCF, might be required for somatic cases",
-                "meta": ["subsample"]
+                "description": "Name to subsample in the multisample VCF, might be required for somatic cases",
```

Comment:
I am making this param more general now, maybe it can stay as "Name to subsample in the multisample VCF"

Reply:
You can choose what to do with this :p